src/components/Table/Table.tsx

```diff
@@ -79,10 +79,6 @@ export const Table: ChakraComponent<
         useRowHeaders,
       });
 
-      // Based on --nypl-breakpoint-medium
-      const breakpointMedium = 600;
-      const windowDimensions = useWindowSize();
```

Comment:
I wonder why I did it this way.  I guess I was just not aware of the breakpoint hooks.